README.md

```diff
@@ -225,8 +232,7 @@ As you see `places` is empty, but the code is changed: there is no `hi` variable
 
 #### No fix
 
-From the first day `putout` was developed with ability to split main process into two concepts: `find` and `fix`.
-This conception mirrorod in the code, so you can find all places with redundant variables without transformation:
+With `fix` turned off, you can find places with redundant variables without making changes to the source file:
```

Comment:
This is not quite right,  splitting main process into two parts is very important information, and `removing redundant variables` is just an example

Reply:
Everything else is awesome :)! Please correct this one and we can merge.